Formula/pocl.rb

```diff
@@ -0,0 +1,33 @@
+class Pocl < Formula
+  desc "Portable Computing Language"
+  homepage "http://portablecl.org"
+  url "http://portablecl.org/downloads/pocl-3.0.tar.gz"
+  sha256 "a3fd3889ef7854b90b8e4c7899c5de48b7494bf770e39fba5ad268a5cbcc719d"
+  license "MIT"
+  head "https://github.com/pocl/pocl.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "opencl-headers" => :build
+  depends_on "pkg-config" => :build
+  depends_on "hwloc"
+  depends_on "llvm"
+  depends_on "ocl-icd"
+
+  fails_with gcc: "5" # LLVM is built with GCC
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args,
+                    "-DCMAKE_INSTALL_RPATH=#{lib}/pocl",
+                    "-DKERNELLIB_HOST_CPU_VARIANTS=distro",
+                    "-DPOCL_INSTALL_ICD_VENDORDIR=#{etc}/OpenCL/vendors"
```

Comment:
As discussed in `ocl-icd` PR, we will need to fix the ICD file before merge.

Could save into `#{libexec}/...` or `#{prefix}/etc/...` and then symlink into `#{etc}/OpenCL/vendors`

Reply:
Oh yeah I forgot about that.  I think that `#{prefix}/etc` makes more sense than `libexec` since it's just a text config file.